src/main/java/duke/tasklist/TaskList.java

```diff
@@ -0,0 +1,155 @@
+package duke.tasklist;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import duke.dukeexception.DukeException;
+import duke.dukeexception.DukeTaskNotFound;
+import duke.task.Task;
+
+/**
+ * A List that is used to contain the Tasks inputted by the user.
+ */
+public class TaskList {
+    private ArrayList<Task> list;
+
+    /**
+     * Constructs an empty TaskList.
+     */
+    public TaskList() {
+        list = new ArrayList<>();
+    }
+
+    /**
+     * Constructs a TaskList with the given tasks.
+     * @param saveFile An ArrayList containing the Tasks from previous usage
+     */
+    public TaskList(ArrayList<Task> saveFile) {
+        list = saveFile;
+    }
+
+    /**
+     * Returns a string to be printed which indicates that the adding of the new Task to the TaskList was successful.
+     * Adds the new task into the list as well.
+     * @param newTask The task to be added into the TaskList
+     * @return The string to be printed that indicates add was successful
+     */
+    public String add(Task newTask) {
+        list.add(newTask);
+        return "Got it. I've added this task:" + System.lineSeparator() + newTask.taskInfo() + listStatus();
+    }
+
+    /**
+     * Returns a string to be printed which contains all the String representation of the tasks in the TaskList.
+     * @return a string representation of all the task in the TaskList.
+     */
+    public String list() {
```

Comment:
Since there is a list interface already in Java, should this method name be changed to prevent confusion?